database/migrations/2021_05_15_121109_create_travel_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTravelTable extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('travel', static function (Blueprint $table): void {
+            $table->id();
+            $table->string('name');
+            $table->string('destination');
+            $table->unsignedInteger('primary_contact_user_id');
+            $table->date('departure_date');
+            $table->date('return_date');
+            $table->unsignedInteger('fee_amount');
```

Comment:
Should probably allow decimal places here since it's currency

Reply:
I thought about that, but in practice I don't think that is useful.